src/Model/File.php

```diff
@@ -124,4 +134,24 @@ public function createFromPath(string $path, string $fileName = null): Model
 
         return $entity;
     }
+
+    /**
+     * Useful method to clean up all draft files.
+     * Can be called as user action or on schedule bases to clean up filestore repository.
+     *
+     * @return $this
+     */
+    public function cleanupDrafts(): Model
```

Comment:
return void

Reply:
why we can't return $this here for chaining?

Reply:
Is this method expected to be chained? No.

Reply:
ok, probably you're right.